scripts/e2e.js

```diff
@@ -15,21 +15,14 @@ const configFile = path.join(Paths.e2eRootDir, 'jest.config.js')
 
 let parentArgs = process.argv.slice(2)
 if (parentArgs.includes('--coverage')) {
-  logger.warn(
-    'Coverages cannot be activated for e2e tests (but can in each e2e test).'
-  )
+  logger.warn('Coverages cannot be activated for e2e tests (but can in each e2e test).')
   parentArgs = parentArgs.filter(a => a !== '--coverage')
 }
 if (!parentArgs.includes('--runInBand')) parentArgs.push('--runInBand')
 const prepareOnly = parentArgs.includes('--prepareOnly')
 
-// eslint-disable-next-line no-unused-vars
-function enableOptimizations() {
```

Comment:
This function did not seem to do anything so i removed it instead of prevent warning from eslint. It is already duplicated here: https://github.com/kulshekhar/ts-jest/blob/4c973400f367ff555449a87ee66843e2c0b7debb/e2e/__helpers__/test-case/utils.ts#L51